recipes/biobb_flexdyn/meta.yaml

```
@@ -43,10 +40,12 @@ test:
 
 about:
   home: https://github.com/bioexcel/biobb_flexdyn
-  license: Apache Software License
+  license: Apache-2.0
   license_family: APACHE
-  summary: Biobb_flexdyn is a BioBB category for studies on the conformational landscape of native proteins.
-  description: "# biobb_flexdyn\n\n### Introduction\nBiobb_flexdyn allows the calculation of protein conformational transitions using the GOdMD package. \nBiobb (BioExcel building blocks) packages are Python building blocks that\ncreate\
+  license_file: LICENSE
+  dev_url: https://github.com/bioexcel/biobb_flexdyn
+  summary: "Biobb_flexdyn is a BioBB category for studies on the conformational landscape of native proteins."
+  description: "# biobb_flexdyn\n\n### Introduction\nBiobb_flexdyn allows the calculation of protein conformational transitions using the GOdMD package. \nBiobb (BioExcel building blocks) packages are Python building blocks that\ncreate\ 
```

yamllint

[error] trailing spaces

(trailing-spaces)